models.go

```diff
@@ -64,52 +105,52 @@ type AlbyMe struct {
 }
 
 type User struct {
-	ID               uint
-	AlbyIdentifier   string `validate:"required"`
-	AccessToken      string `validate:"required"`
-	RefreshToken     string `validate:"required"`
-	Email            string
-	Expiry           time.Time
-	LightningAddress string
-	Apps             []App
-	CreatedAt        time.Time
-	UpdatedAt        time.Time
+	ID               uint      `json:"id"`
```

Comment:
I think we should not return the database models directly. I think we should undo these changes and make new API models.